roottest/cmake/modules/RoottestMacros.cmake

```diff
@@ -683,7 +683,7 @@ macro(ROOTTEST_SETUP_MACROTEST)
                -e "gROOT->SetMacroPath(\"${CMAKE_CURRENT_SOURCE_DIR}\")"
                -e "gInterpreter->AddIncludePath(\"-I${CMAKE_CURRENT_BINARY_DIR}\")"
                -e "gSystem->AddIncludePath(\"-I${CMAKE_CURRENT_BINARY_DIR}\")"
-               ${RootExternalIncludes} ${RootExeOptions}
+               ${RootExternalIncludes} ${RootExeOptions} ${ARG_ROOTEXE_OPTS}
```

Comment:
The variable `RootExeOptions` and the argument `ROOTEXE_OPTS` now seems to have the exact same semantic.  Any reason to keep both?

Reply:
This will be handled in follow-up PR